index.html

```diff
@@ -18,10 +18,67 @@
 <body>
   <header>
     <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Code Journal</h1>
+        </div>
+      </div>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div class="row">
+      <div data-view="entry-form">
+        <form>
+          <div class="row">
+            <div class="column-full">
+              <h2>New Entry</h2>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <img class="display" src="images/placeholder-image-square.jpg">
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <label for="title">Title</label>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <input required type="text" name="title" id="title">
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <label for="photourl">Photo URL</label>
+            </div>
+          </div>
```

Comment:
I think you have too many `row`s and `column-half`s here
When in a desktop browser, the Title and PhotoURL inputs should be on the right side of the image.

Doing so would mean that the image, title input, and photo URL input should be in the same `row`. And the title and photo URL inputs should be inside a `column-half` so that they are aligned to the right of the image since the image will still be in its own `column-half` in the same row.